cypress-tests/cypress/e2e/v2/projectBasics.cy.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  getRandomString,
+  validateLoginV2,
+} from "../../support/commands/general";
+import { ProjectIdentifierV2 } from "../../support/types/project.types";
+import { User } from "../../support/types/user.types";
+import {
+  deleteProjectFromAPIV2,
+  getProjectByNamespaceAPIV2,
+} from "../../support/utils/projectsV2.utils";
+
+const sessionId = ["projectBasics", getRandomString()];
+
+beforeEach(() => {
+  // Restore the session (login)
+  cy.session(
+    sessionId,
+    () => {
+      cy.robustLogin();
+    },
+    validateLoginV2
+  );
+});
+
+describe("Project - create, edit and delete", () => {
+  // Define some project details
+  const projectNameRandomPart = getRandomString();
+  const projectName = `project/$test-${projectNameRandomPart}`;
+  const projectPath = `project-test-${projectNameRandomPart}`;
+  const projectDescription = "This is a test project from Cypress";
+  const projectIdentifier: ProjectIdentifierV2 = {
+    slug: projectPath,
+    id: null,
+    namespace: null,
+  };
+
+  // Cleanup the project after the test -- useful on failure
+  after(() => {
+    getProjectByNamespaceAPIV2(projectIdentifier).then((response) => {
+      if (response.status === 200) {
+        projectIdentifier.id = response.body.id;
+        projectIdentifier.namespace = response.body.namespace;
+        deleteProjectFromAPIV2(projectIdentifier);
+      }
+    });
+  });
+
+  it("Project - create, edit and delete", () => {
+    // Create a new project
+    cy.visit("/v2");
+    cy.getUserData().then((user: User) => {
+      const username = user.username;
+      cy.get("#plus-dropdown").should("exist").click();
+      cy.getDataCy("navbar-project-new").click();
+      cy.location("hash").should("eq", "#create-project");
+      cy.getDataCy("new-project-modal").should("exist");
```

Comment:
Add a test to check that the modal (or modal header) contains "Create a new project".

Reply:
I would say it's a similar case of https://github.com/SwissDataScienceCenter/renku/pull/3881/files#r1934094160
The idea behind using `getDataCy` is to disentangle getting the proper element from matching some specific text that might easily change in a future UI release.